frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -568,21 +470,24 @@ const ManagePolicyPage = (managePoliciesPageProps: {
         )}
         {showInheritedPoliciesButton && showInheritedPolicies && (
           <div className={`${baseClass}__inherited-policies-table`}>
-            <PoliciesListWrapper
-              isLoading={
-                isLoadingGlobalPolicies && isLoadingFailingPoliciesWebhook
-              }
-              policiesList={globalPolicies}
-              onRemovePoliciesClick={noop}
-              resultsTitle="policies"
-              canAddOrRemovePolicy={canAddOrRemovePolicy(
-                currentUser,
-                selectedTeamId
-              )}
-              tableType="inheritedPolicies"
-              selectedTeamData={selectedTeamData}
-              currentAutomatedPolicies={currentAutomatedPolicies}
-            />
+            {globalPoliciesError && <TableDataError />}
+            {!globalPoliciesError &&
+              (globalPolicies === undefined ? (
```

Comment:
Could we switch the ternary conditions around and do `(globalPolicies ? ( <PoliciesListWrapper> ) : ( <Spinner />)`?